docs/getting_started.md

```diff
@@ -36,7 +36,7 @@ components:
 -   [Doxygen](http://doxygen.org) >= 1.8.15 for the documentation
 -   [Geant4](http://geant4.org/) for some examples
 -   [HepMC](https://gitlab.cern.ch/hepmc/HepMC3) >= 3.2.1 for some examples
--   [Intel Threading Building Blocks](https://01.org/tbb) >= 2020.1 for the examples
```

Comment:
https://01.org/tbb seems to be up

Reply:
I think it was only getting 403s from Microsoft data centers.